clustered/client/src/test/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactoryTest.java

```diff
@@ -69,14 +70,12 @@ public void testCreate() throws Exception {
     EhcacheClientEntityFactory factory = new EhcacheClientEntityFactory(connection);
     factory.create("test", null);
     verify(entityRef).create(any(UUID.class));
```

Comment:
Wondering how that verify worked. Shouldn't the `create` be verfied on `any(ClusteredTierManagerConfiguration.class)`?